src/Tribe/Subscribe/Purchase.php

```diff
@@ -18,30 +19,63 @@ class Purchase {
 	 *
 	 */
 	public function hook() {
-		add_action( 'event_ticket_woo_attendee_created', [ $this, 'connect' ], 10, 4 );
+
+		add_action( 'event_ticket_woo_attendee_created', [ $this, 'woo_timeline' ], 10, 4 );
+		add_action( 'event_ticket_woo_attendee_created', [ $this, 'woo_subscribe' ], 10, 4 );
```

Comment:
Does it matter what order they are done in? Should we establish separate priority to hook into?

Reply:
It does not matter as they are two different HubSpot calls and in Sprint 4 they will be hooked into the Queue System.